### PR TITLE
Post: Lifehacking 101: I'm bad at to-do lists, now what?

### PR DESCRIPTION
**Pull request by mauroservienti**
- [x] Synopsis
- [x] Header image https://unsplash.com/photos/2FaCKyEEtis
- [x] Header image credits
- [x] Review
- [x] Rename file to match publish date
- [ ] Buffer content

### CODE REVIEW

File: _posts/bad-at-to-do-lists-now-what.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Running on zero-notifications is a relief; with no pressure coming from badges or any other sort of distracting and pressure-designed crap, my day is already doing great.
+
+However, as already said in the above-linked article, the day in Particular Software is a bit particular (no pun intended). Time zones, multiple task forces, wearing more than one hat ranging from "product management" to support, all participate in the risk of making the day a bit of a mess. 
+
+## Please welcome context switching 
+
+One of the side effects of wearing multiple hats throughout the day is the rise of context switching. Let's do some extreme math to put the issue in context; if we were to have only 5-minute long tasks and an 8-hour workday, the hypothetical result is 96 completed tasks: 12 tasks every hour, times 8 hours. Now, factor in multiple hats, and again for the sake of the sample, let's imagine that each assignment requires a different hat and that to change it (context switch), it takes our brain about 5 minutes. In such a scenario, we'll be able to complete 50% of the 96 tasks a day—a colossal cutoff.
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:29]:
```suggestion
One of the side effects of wearing multiple hats throughout the day is the rise of context switching. Let's do some extreme math to put the issue in context; if we were to have only 5-minute long tasks and an 8-hour workday, the hypothetical result is 96 completed tasks: 12 tasks every hour, times 8 hours. Now, factor in multiple hats, and again for the sake of the sample, let's imagine that each assignment requires a different hat and that to change it (context switch), it takes our brain about 5 minutes. In this scenario, we'll be able to complete 50% of the 96 tasks each day — a colossal dropoff.
```

File: _posts/bad-at-to-do-lists-now-what.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+One could argue a zero-context switching approach would be an ideal world. Probably. I guess it depends on the person and the task. There are things that I enjoy a lot, like writing code or writing in general, that bores me very quickly; after a couple of hours in the best case, my mind starts digressing, signaling it's time for a change.
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:31]: > There are things that I enjoy a lot, like writing code or writing in general, that bores me very quickly

This looks contradictory. Do you mean you enjoy writing code but that writing in general bores you?

File: _posts/bad-at-to-do-lists-now-what.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+> If there is an unsolved problem, like a bug or a not yet wholly identified solution, that's another story.
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:33]:
```suggestion
> If there is an unsolved problem, like a bug or a not-yet wholly-identified solution, that's another story.
```

File: _posts/bad-at-to-do-lists-now-what.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+I also enjoy many diverse activities ranging from solving technical problems to "product management" to more business-related activities,
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:35]:
```suggestion
I also enjoy many diverse activities ranging from solving technical problems to "product management" to more business-related activities.
```

Combine this with the next paragraph

File: _posts/bad-at-to-do-lists-now-what.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+However, if I follow my brain's hint and switch to another activity, the effort required is extreme. In retrospect, I can state that the first 15 to 20 minutes of the next task are "wasted" cycles in the attempt to clear up my mind and get the necessary concentration to attack the new problem.
+
+> On a side note, that is the reason why I love "commuting" on foot. My office is precisely a mile from home. In the best case, it's an 8-minute walk (I'm in a hurry sometimes) or a 15-minute relaxing one. Either one is the perfect decompression time. By the time I get home, my brain is ready for the 3-year old little Dracula ;-)
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:39]:
```suggestion
> On a side note, that is the reason why I love "commuting" on foot. My office is precisely a mile from home. That's a 15-minute leisurely walk or an 8-minute brisk walk. Either one is a good decompression time. By the time I get home, my brain is ready for the 3-year old little Dracula ;-)
```

File: _posts/bad-at-to-do-lists-now-what.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+## Fillers
+
+Since a couple of months, so it's probably too early for some conclusions, I started injecting filler activities between daily tasks. One of the things I enjoy is reading.
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:43]:
```suggestion
For a couple of months, so it's probably too early for some conclusions, I started injecting filler activities between daily tasks. One of the things I enjoy is reading.
```

Consider expanding on what you mean here by "filler" activity. For example:

```suggestion
For a couple of months, so it's probably too early for some conclusions, I started injecting filler activities between daily tasks. These are leisure activities that I enjoy doing and that I can pick up and put down in short bursts. That is, they don't require a lot of mental activity to start or to stop.
```

File: _posts/bad-at-to-do-lists-now-what.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+Reading turned out to be an excellent filler activity. Building LEGO is another satisfactory one.
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:45]: If you accept my second suggestion above, combine this sentence with the previous paragraph:

```suggestion
For me, reading is an excellent filler activity. Building LEGO is another satisfactory one.
```

File: _posts/bad-at-to-do-lists-now-what.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+I extensively use [Pocket](https://getpocket.com) as the tool to keep track of all the articles I want to read. I carefully tag articles that I continuously store. I tend to read about everything except technology; if you could look at my tags, very few items are labeled as `technology` or `architecture`. My reading interests are mostly politics, economics, mindfulness, psychology, and art.
+
+Whenever I need to switch to a different task, I merely open Pocket and select a couple or more articles to read, not more than 10 minutes or even less. 
+I keep an eye on the topics; I carefully select topics that are very distant from what I was doing right before and from what I'm going to do next. Results are dual:
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:50]:
```suggestion
I keep an eye on the topics; I carefully select topics that are very distant from what I was doing right before and from what I'm going to do next. The result is two-fold:
```

File: _posts/bad-at-to-do-lists-now-what.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+- I feed my soul (typical Italian idiom that has nothing to do with religion in this case),
+- I clear my brain from the leftovers of the previous task and get it ready for the next.
+
+## Conclusion
+
+Context switching is a problem, that's no doubt, mainly if it plays a primary role in our days. On the one hand, it's essential to reduce context switching to the minimum required; on the other hand, we can benefit from it, at least I did. It comes with a positive effect for people like me that need to have variety throughout their day.
```
Comment (kbaley) [inline _posts/bad-at-to-do-lists-now-what.md:57]:
```suggestion
There's no doubt context switching is a problem and that it plays a key role in our days. On the one hand, it's essential to reduce context switching to the minimum required; on the other hand, we can benefit from it, at least I did. It comes with a positive effect for people like me that need to have variety throughout their day.
```

File: _posts/bad-at-to-do-lists-now-what.md (hunk 2 above)
Comment (mauroservienti) [reply _posts/bad-at-to-do-lists-now-what.md:31]: I mean enjoy both of them, but both of them bore me very quickly. So, I wanna write code but not for 8 hours a day

File: _posts/bad-at-to-do-lists-now-what.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+## Please welcome context switching 
+
+One of the side effects of wearing multiple hats throughout the day is the rise of context switching. Let's do some extreme math to put the issue in context; if we were to have only 5-minute long tasks and an 8-hour workday, the hypothetical result is 96 completed tasks: 12 tasks every hour, times 8 hours. Now, factor in multiple hats, and again for the sake of the sample, let's imagine that each assignment requires a different hat and that to change it (context switch), it takes our brain about 5 minutes. In this scenario, we'll be able to complete 50% of the 96 tasks each day — a colossal dropoff.
+
+One could argue a zero-context switching approach would be an ideal world. Probably. I guess it depends on the person and the task. There are things that I enjoy a lot, like writing code or writing in general, that bores me very quickly; after a couple of hours in the best case, my mind starts digressing, signaling it's time for a change.
```
Comment (mauroservienti) [inline _posts/bad-at-to-do-lists-now-what.md:31]: @kbaley what about something like:

```suggestion
One could argue a zero-context switching approach would be an ideal world. Probably. I guess it depends on the person and the task. For example, when it comes to me, there are things, like writing or programming, that I enjoy a lot. Simultaneously, the excitement starts to sunset, usually in a couple of hours. My mind starts digressing, signaling it's time for a change.
```

File: _posts/bad-at-to-do-lists-now-what.md (hunk 2 above)
Comment (kbaley) [reply _posts/bad-at-to-do-lists-now-what.md]: In that case:

> There are things that I enjoy a lot but that bore me very quickly, like writing code or writing in general.

**Pull request merged**